index.bs

```diff
@@ -583,36 +583,33 @@ interface ML {
 </script>
 
 The {{ML/createContext()}} method steps are:
-1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, then throw a "{{SecurityError!!exception}}" {{DOMException}} and abort these steps.
 1. Let |promise| be [=a new promise=].
+1. Return |promise| and run the following steps [=in parallel=].
```

Comment:
I'd suggest indent the "in parallel" steps that follow so the in parallel steps start from index 1.

I'm used to seeing the "Return _promise_." on its own line after the "in parallel" substeps. Reading the [in parallel definition](https://html.spec.whatwg.org/multipage/infrastructure.html#in-parallel) I don't think there's any normative impact either way so we can return early too if that is a more common practice.

If we want to return the promise on the same line with "run the following steps in parallel", we may want to update [`compute()`](https://www.w3.org/TR/webnn/#dom-mlcontext-compute) similarly.

Maybe return promise on its own line maps more directly with the code that is written to implement this (also for tests) and thus less error prone?

Reply:
No problem, I can use the prose used in Web Bluetooth, Compute Pressure etc, or the prose used in WebRTC and others.

> the method when invoked, MUST return [a new promise](https://webidl.spec.whatwg.org/#a-new-promise) promise and run the following steps [in parallel](https://html.spec.whatwg.org/multipage/infrastructure.html#in-parallel): 

or,

> - If [condition] is true, return a promise [rejected](https://w3c.github.io/webrtc-pc/#dfn-reject) with a newly created WhateverError.
> - (...) other steps
> - Let p be a new promise.
> - (all the other steps)
> - Return p.

I have mostly used/preferred the former, since it gives a better hunch on [what's happening](https://jakearchibald.com/2015/tasks-microtasks-queues-and-schedules/) in the event/task/microtask loop. The method really returns and the rest happen on a new timeline that implements "parallel" on the given underlying platform.

Reply:
I'll let the editors @huningxin @wchao1115 to share which convention they'd prefer to use.